### DISCUSSION TITLE
更方便好看的檔案總管

### DISCUSSION THREAD

**Discussion by sys511613** [1101-GS4719-C/Python-Community-GS4719-C#643] [General]
在寫程式或是使用電腦的時候，常常會需要將檔案在多個資料夾中進行處理，如果用windows內建的檔案總管的話就需要開很多個視窗
![image](https://user-images.githubusercontent.com/73024991/170988153-5ea4cde6-b839-4af9-94b6-a21473bf7fc8.png)
⬆️像這樣

這樣下來常常會找不到哪一個視窗才是自己想要用的，也很佔位置
所以介紹一些更方便的程式，可以取代內建的檔案總管！

## Files App
[下載連結 ](https://www.microsoft.com/store/productId/9NGHP3DX8HDX)
這是一個免費程式，打開後介面就跟原本的檔案總管一樣，不同的是他可以像是瀏覽器的分頁那樣開啟資料夾
![image](https://user-images.githubusercontent.com/73024991/170993697-6eff440f-e9f6-4e60-b7de-b5607ed17ff0.png)
還可以把兩個資料夾並排打開，這樣檔案移動可以方便很多
![image](https://user-images.githubusercontent.com/73024991/170990524-5866fc6f-a2b7-4703-b907-c384128d0bd6.png)
### 注意
這個程式還有很多有趣的功能可以去玩玩看(像是Bundles或資料夾顏色都很方便)
雖然好用但會有一些小bug，開啟資料夾的速度也比內建的檔案總管慢，使用前請詳閱公開說明書

## Shrestha Files Pro X
[下載連結](https://www.microsoft.com/store/productId/9NPNFFSV2HQM)
這個程式原價339，現在有特價169(偶爾會出現限時免費)
這個程式比Files App功能更多，開檔案的速度也比Files Apps快，也可以設定很多佈景主題
當然分頁、資料夾並排顯示的功能也都有
![image](https://user-images.githubusercontent.com/73024991/170993292-5d585116-71c3-428a-b38e-d2945256a39c.png)

不過私心還是覺得Files App的icon和介面比較好看
所以現在都是內建的檔案總管跟Files App交替使用中
如果受夠內建檔案總管的一大堆視窗的話可以考慮看看

**Comment by d26488399** [comment]
感謝分享，希望微軟也能趕快加入分頁功能

**Comment by jason79461385** [comment]
看起來還不錯!像平常我都要用兩個視窗後再整理東西，看來使用這個方便很多。

**Comment by JSHT** [comment]
Shrestha Files Pro X 可以自定義側邊 icon 連結路徑，這在 windows 內建檔案總管也有類似功能。
只要在任意資料夾點擊右鍵、選擇釘選到快速存取，就能將資料夾路徑弄到側邊欄
![image](https://user-images.githubusercontent.com/48956859/171091357-7ea63801-445a-4c9b-8849-3b19ba72b050.png)
![image](https://user-images.githubusercontent.com/48956859/171091328-3130181a-0644-400e-8761-81b7c8183145.png)

**Comment by awkward-willy** [comment]
個人還是比較習慣用原本的檔案總管，比較直覺跟快速
不過開很多的情況下真的得靠多次 Alt + tab 來找
上面也有人提到快速存取，我自己也常會把正在使用的資料夾釘選
使用起來真的會方便不少

不過這也是相當實用的工具，希望未來能持續優化跟普及

**Comment by ghost** [comment]
看起來還不錯 雖然不確定對效能有沒有影響 不過原本的用習慣 應該就不會考慮了

**Comment by Rosancl** [comment]
感謝，很實用！這幾天寫專題也開了很多資料夾，太多視窗真的會感到困擾。

**Comment by bloodtrail0511** [comment]
感謝!每次檔案總管都會開超多個超亂，總算有一個好辦法解決了

**Comment by yucheng1028** [comment]
感覺不錯 載來用看看

**Comment by stan890425** [comment]
在整理檔案的時候一定很好用~~

**Comment by Yuhsin000** [comment]
終於不用一直喬視窗大小了

**Comment by Ben6uo** [comment]
這個真的太方便了 以後就知道那個是那個了

**Comment by leon10456** [comment]
不管什麼領域AI運算都很重要呢

**Comment by 110409538** [comment]
感覺很實用,之後會試試看.

**Comment by etin2468** [comment]
好酷喔，對我這種眼殘的人來說真有幫助

**Comment by Lynn12378** [comment]
非常有幫助，每次手殘都不小心按成關閉視窗，用分頁應該比較不會按到
